course_info/icommons.py

```diff
@@ -122,7 +121,7 @@ def _course_info_instructor_list(self, course_instance_id, **kwargs):
                 instructors = []
                 for person in rv['results']:
                     course_person_schema(person)
-                    if person.get('role')['role_id'] in INSTRUCTOR_ROLE_IDS:
+                    if person.get('role') and person.get('role')['role_id'] in INSTRUCTOR_ROLE_IDS:
```

Comment:
That's bizarre... it works for me. Unless `person` is somehow `None`, but the `for` loop shouldn't give you a `NoneType` object I wouldn't think.